.github/workflows/_test_pax.yaml

```diff
@@ -13,10 +13,20 @@ on:
         description: Extra command line args to pass to test-pax.sh
         default: ""
         required: false
+      BADGE_FILENAME:
+        type: string
+        description: 'Name of the endpoint JSON file for shields.io badge'
+        required: false
+        default: 'badge-pax-mgmn-test'
+      ARTIFACT_NAME:
+        type: string
+        description: 'Name of the artifact zip file'
+        required: false
+        default: 'artifact-pax-mgmn-test'
     outputs:
       TEST_STATUS:
         description: 'Summary pass/fail value indicating if results from tests are acceptable'
-        value: ${{ jobs.publish-test.outputs.STATUS }}
+        value: ${{ jobs.sitrep.outputs.STATUS }}
```

Comment:
This is similar to our current practice, which uses a final postprocessing job to determine whether the overall MGMN test suite succeeds.

It has the problem that error feedback is delayed and made it difficult to investigate which individual tests failed.

For the new sitrep reporting system, let's do the following:
- Individual jobs should fail immediately if the tests that it ran do not succeed. However, it should collect and generate test artifacts regardless of whether the tests pass or fail. [The `continue-on-error` option can be helpful here](https://github.com/NVIDIA/JAX-Toolbox/blob/1cec8a38ecc0ceb7f63b30ced4480910192fee7b/.github/workflows/_test_jax.yaml#L65).
- The sitrep/postprocessing step should then follow regardless of the success/fail status of individual test jobs.

This helps to locate the error status for easier debugging and tracking.

Reply:
For the MGMN usecase, the tests are submitted to slurm via SSH. To mark an individual job as failure I can inspect the exit code and mark it accordingly. We would still need the overall status though, if there's any other recommended way to get this output, I'm happy to incorporate that.

Reply:
Makes sense. How about we make individual jobs fail while also let the overall status be the Boolean AND of the individual job states?

Reply:
Can do, but then we won't need `continue-on-error` I believe.

Reply:
Also could you please add a natural language description of the logic regarding 
> The STATUS has to be derived not only from the job result, but also from its metrics
so that it serves as a form of documentation?

Reply:
> At face value, the specific [error message](https://github.com/NVIDIA/JAX-Toolbox/actions/runs/7053172928/job/19199718748#step:8:18) from the job that you provided looks coincidental. I understand that it may be the result of the true underlying error, but the message itself does not clearly indicate that. Could we improve on it?

Not sure if there's a way to provide a more informative error based on the failure, might have to look at it outside the scope of this PR.

Reply:
> Also could you please add a natural language description of the logic regarding
> 
> > The STATUS has to be derived not only from the job result, but also from its metrics
> 
> so that it serves as a form of documentation?

Yes will do it in the github actions yml.

Reply:
Done in https://github.com/NVIDIA/JAX-Toolbox/pull/401/commits/40aebf593c3e169b516f43ec3f429c327d2ec5e1